### DISCUSSION TITLE
Netlify Deployment

### DISCUSSION THREAD

**Discussion by mariosconsta** [courses-ionio/hci#1696] [Announcements]
Βλέπω σε αρκετά Pull Request πως δεν υπάρχουν link για το Netlify. Αν κάποιος δεν ξέρει πως να το κάνει setup, ίσος μπορώ να βοηθήσω!

**Comment by epidrome** [comment]
γιατί δεν κάνεις ένα πολύ σύντομο ντέμο με οθόνες ή και γκιφάκια να το ποστάρεις στο https://github.com/courses-ionio/hci/discussions/categories/show-and-tell
